airflow/ui/src/components/TriggerDag/TriggerDAGForm.tsx

```diff
@@ -214,52 +221,43 @@ const TriggerDAGForm: React.FC<TriggerDAGFormProps> = ({
                       extensions={[json()]}
                       height="200px"
                       onBlur={() => {
-                        const prettifiedJson = validateAndPrettifyJson(
-                          field.value,
-                        );
-
-                        field.onChange(prettifiedJson);
+                        field.onChange(validateAndPrettifyJson(field.value));
                       }}
                       style={{
                         border: "1px solid #CBD5E0",
                         borderRadius: "8px",
                         outline: "none",
                         padding: "2px",
+                        width: "100%",
                       }}
                       theme={colorMode === "dark" ? githubDark : githubLight}
                     />
-                    {Boolean(errors.conf) && (
-                      <Text color="red.500" fontSize="sm" mt={2}>
-                        {errors.conf}
-                      </Text>
-                    )}
-                  </Box>
+                    {Boolean(errors.conf) ? (
+                      <Field.ErrorText>{errors.conf}</Field.ErrorText>
+                    ) : undefined}
+                  </Field.Root>
                 )}
               />
 
-              <Text fontSize="md" mb={2} mt={6}>
-                Dag Run Notes
-              </Text>
               <Controller
                 control={control}
                 name="note"
                 render={({ field }) => (
-                  <Input {...field} placeholder="Optional" size="sm" />
+                  <Field.Root mt={6}>
+                    <Field.Label fontSize="md">Dag Run Notes</Field.Label>
+                    <Input {...field} placeholder="Optional" size="sm" />
+                  </Field.Root>
                 )}
               />
             </Box>
           </Accordion.ItemContent>
         </Accordion.Item>
       </Accordion.Root>
-      {Boolean(errors.date) && (
-        <Text color="red.500" fontSize="sm" mt={2}>
-          {errors.date}
-        </Text>
-      )}
+      <ErrorAlert error={errors.date ?? errorTrigger} />
       <Box as="footer" display="flex" justifyContent="flex-end" mt={4}>
         <HStack w="full">
           {isDirty ? (
-            <Button onClick={() => reset()} variant="outline">
+            <Button onClick={handleReset} variant="outline">
```

Comment:
I did a manual test and noticed that the reset button does not reset the "Data interval start/end date" fields - is this by intent or a bug?

Reply:
No, this shouldn't be the case.
It's weird, I can't reproduce this in local.
which browser are you using?

Reply:
![image](https://github.com/user-attachments/assets/ac7989e5-bdd6-4ff0-9f7c-6a0c0dfc53c4)

Reply:
Oh, yeah, can confirm. All this is not happening on Chrome.

Reply:
@bbovenzi I remember we had problems with Firefox and the datetime selector in the past - do you recall how we fixed this in the legacy UI? Can we transfer the same approach to the new UI as well (in a different PR)?